common/src/main/java/tachyon/network/protocol/databuffer/DataBuffer.java

```diff
@@ -42,4 +42,11 @@
    * @return a read-only ByteBuffer representing the data
    */
   public abstract ByteBuffer getReadOnlyByteBuffer();
+
+  /**
+  * Release the underlying buffer of this DataBuffer if no longer needed.
+  *
+  * @return True if the underlying buffer is released or no need to release
+  */
+  public abstract boolean release();
```

Comment:
So if `ByteBuf.release()` is called when its reference count is not one, it returns `false`. I think it would be useful to return boolean because sometimes we might want to check that. (On the other hand, we might not want to throw exception when `ByteBuf.release()` returns false, because this can happen during certain unit tests.